lib/Agrammon/OutputFormatter/GUI.pm6

```diff
@@ -25,7 +25,7 @@ sub get-data($model, $outputs) {
             for sorted-kv($_) -> $output, $raw-value {
                 my $format = $model.output-format($module, $output);
                 my $value = flat-value($raw-value);
-                my $formattedValue = $format ?? sprintf($format, $value)
+                my $formattedValue = ($format && $value) ?? sprintf($format, $value)
```

Comment:
Do we never want to format `0`s? (If we do, maybe it should be `&& $value.defined`

Reply:
Indeed, 0 is certainly a valid value. Fixed.